indexing.rdf

```diff
@@ -5,6 +5,11 @@
      xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
      xmlns:dcterms="http://purl.org/dc/terms/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+    <owl:Ontology rdf:about="http://fedora.info/definitions/v4/indexing#">
```

Comment:
Also, the owl:Ontology section mostly duplicates the following rdf:Description section -- please either remove the owl:Ontology  node or the rdf:Description node, making sure that the relevant dcterms:* properties are included.